server.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const app = express();
+const mongoose = require("mongoose");
+const User = require("./models/User");
+const bodyParser = require("body-parser");
+require("dotenv").config();
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+//Connect Mongoose Database
+mongoose.connect(process.env.DB_CONNECTION_STRING).then(() => {
+  console.log("Connected to database");
+});
+
+//Get all users
+app.get("/users", (req, res) => {
+  User.find()
+    .then((users) => {
+      res.json(users);
+    })
+    .catch((error) => {
+      res.status(500).json({ message: error.message });
+    });
+});
+
+// Add a new user
+app.post("/users", (req, res) => {
+  const user = new User({
+    name: req.body.name,
+    email: req.body.email,
+    age: req.body.age,
+  });
+  user
+    .save()
+    .then((newUser) => {
+      res.status(201).json(newUser);
+    })
+    .catch((error) => {
+      res.status(400).json({ message: error.message });
+    });
+});
+
+// Update a user by ID
+app.put("/users/:id", (req, res) => {
+  User.findById(req.params.id)
+    .then((user) => {
+      if (!user) {
+        return res.status(404).json({ message: "User not found" });
+      }
+      user.name = req.body.name || user.name;
+      user.email = req.body.email || user.email;
+      user.password = req.body.password || user.password;
```

Comment:
Passwords should never be stored as plain text in your database. Take a look at this[ stack overflow question](https://stackoverflow.com/questions/43092071/how-should-i-store-salts-and-passwords-in-mongodb).